.github/labeler.yml

```diff
@@ -0,0 +1,50 @@
+GHA:
+  - .github/**/*
+
+area/config:
+  - charts/registry/config/**/*
+
+documentation:
```

Comment:
Perhaps we could rename the existing label to also be `area/docs`, but we can do in a follow-up if you prefer.

Reply:
Updated